scripts/qa_crowbarsetup.sh

```diff
@@ -2007,8 +2007,13 @@ function reboot_nodes_via_ipmi()
                 if ipmitool -H $ip -U root -P $pw power status | grep -q "is off"; then
                     ipmitool -H $ip -U root -P $pw power on
                     sleep $((10 + RANDOM % 15))
+                else
+                    ipmitool -H $ip -U root -P $pw power off
+                    wait_for 5 2 "ipmitool -H $ip -U root -P $pw power status | grep -q 'is on'" "node to power on"
```

Comment:
After testing locally i think it is enough. ipmitool power off/on is like pulling a plug and power status is updated in 2-3 seconds.